src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -861,21 +861,15 @@ class Telemetry : public PluginBase {
  */
 bool operator==(const Telemetry::PositionVelocityNED &lhs,
                 const Telemetry::PositionVelocityNED &rhs);
-/**
- * @brief Equal operator to compare two `Telemetry::Position` objects.
- *
- * @return `true` if items are equal.
- */
 
 bool operator==(const Telemetry::PositionNED &lhs,
```

Comment:
You will probably need to document that, or the CI will complain :slightly_smiling_face:.